prometheus_client/exposition.py

```diff
@@ -102,14 +103,30 @@ def do_GET(self):
     def log_message(self, format, *args):
         """Log nothing."""
 
+    @staticmethod
+    def factory(registry):
```

Comment:
@brian-brazil Dunno if python does dead code elimination, but variables are required to better document the code.